src/api/dirmonitor/polling.c

```diff
@@ -80,7 +79,7 @@ int add_dirmonitor(struct dirmonitor_internal* monitor, const char* path)
 
   // Check if there is any space we can reuse
   for (size_t i = 0; i < monitor->fd_count; ++i) {
-    if (monitor->fds[i].fd == -1)
+    if (monitor->fds[i].fd == -1) {
     {
```

Comment:
err, now this is an issue... thanks github
I'll fix this when I find the time

Reply:
yeah, should've reminded you lol